addons/mail/static/src/messaging/entity_field.js

```diff
@@ -811,31 +905,34 @@ class EntityField {
      *   if null, unlink all items in the relation of provided entity.
      */
     _setRelationUnlinkOne2Many(entity, newValue) {
-        if (!entity.constructor.get(entity)) {
-            // Entity has already been deleted.
-            // (e.g. unlinking one of its reverse relation was causal)
-            return;
-        }
-        const prevValue = this.read(entity);
-        const value = newValue === null
+        // convert newValue to array of localId, null is considered unlink all
+        const otherLocalIds = newValue === null
```

Comment:
I think special value `null` is just used in this file, and the few occurrences could be replaced `'unlink'` with all entities since these unlink commands are used in isolation. Would be nice to remove this once and for all.

Reply:
Would be nice to remove `null` special value for `'unlink'`.